test/placement/library_test.cpp

```diff
@@ -0,0 +1,69 @@
+#include <catch.hpp>
+
+#include <boost/geometry.hpp>
+
+#include <ophidian/placement/Library.h>
+
+using namespace ophidian::placement;
+using namespace ophidian::standard_cell;
+using namespace ophidian::geometry;
+
+namespace {
+class StandardCellsFixture {
+public:
+    Cell cell1, cell2;
+    Pin pin1, pin2;
+
+    StandardCells std_cells;
+
+    StandardCellsFixture() {
+        cell1 = std_cells.add(ophidian::standard_cell::Cell(), "cell1");
+        cell2 = std_cells.add(ophidian::standard_cell::Cell(), "cell2");
+
+        pin1 = std_cells.add(Pin(), "pin1", PinDirection::NOT_ASSIGNED);
+        pin2 = std_cells.add(Pin(), "pin2", PinDirection::NOT_ASSIGNED);
+    }
+};
+
+bool multiBoxComparison(const MultiBox & multiBox1, const MultiBox & multiBox2) {
+    for (auto box1 : multiBox1) {
```

Comment:
I'd change this to` ophidian::geometry::equals(const MultiBox & multiBox1, const MultiBox & multiBox2) `for example.